portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,13 +52,15 @@
   protected DatastoreService datastore;
   protected Gson gson;
   protected UserService user;
+  protected BlobstoreService blobstore;
 
   public DataServlet() {
     super();
 
     datastore = DatastoreServiceFactory.getDatastoreService();
     gson = new Gson();
     user = UserServiceFactory.getUserService();
+    blobstore = BlobstoreServiceFactory.getBlobstoreService();
```

Comment:
This instance of the BlobstoreService is never used (a new one is defined in line 128).